docs/guides/admin/docs/modules/notification.md

```diff
@@ -0,0 +1,37 @@
+# Mattermost Notification
```

Comment:
A short description is missing. What is this operation for?

Reply:
…and the list of configuration options is missing

Reply:
@isabellakutger Just wondering: Why are you actually adding a new WOH for notification at all (instead of just enhancing the existing HTTPNotifyWorkflowOperationHandler)?  At the end of the day, the description suggests that this WOH is not specifically built for Mattermost.